packages/core/src/generators/schema.ts

```diff
@@ -48,7 +44,27 @@ export async function generateSchema(
   config.source.initialize?.(config);
 
   // Invoke the content source resolver to retrieve the content nodes
-  const allContentNodes = await config.source.fetch(config.content);
+  let allContentNodes: Record<string, any> = {};
+
+  const addRecord =
```

Comment:
[nit] could we also return the newly added record? That would make testing of this easy